firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.6
+
+fixed - Record document id in schema view and revert timestamp change (#357)
```

Comment:
fixed - Fixed issue with timestamp values not showing up in the `latest` view (#357)

Reply:
Done.